Tasks/AzureCloudPowerShellDeploymentV1/task.json

```diff
@@ -16,8 +16,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 175,
```

Comment:
Take latest master and update versions of these tasks as well AzurePowerShellV2,AzurePowerShellV3,AzurePowerShellV4,AzurePowerShellV5

Reply:
updating...